README.md

```diff
@@ -7,6 +7,9 @@ These resources include a calendar of tech events, a list of meetup groups,
 local beginner-friendly resources, shout-outs of our gracious sponsors, and
 other ways to get involved in the Lansing tech community.
 
+## Code of Conduct
+All participants are expected to treat others with respect and follow our [Code of Conduct](https://github.com/codeforlansing/codeofconduct#code-for-lansings-code-of-conduct).
```

Comment:
Would you mind updating the URL in this link to this?

https://github.com/lansingcodes/www/blob/master/.github/CODE_OF_CONDUCT.md

Reply:
Updated the URL to https://github.com/lansingcodes/www/blob/master/.github/CODE_OF_CONDUCT.md